examples/ReactNativeTodoList/.gitignore

```diff
@@ -0,0 +1,62 @@
+# OSX
+#
+.DS_Store
+
+# Xcode
+#
+build/
+*.pbxuser
+!default.pbxuser
+*.mode1v3
+!default.mode1v3
+*.mode2v3
+!default.mode2v3
+*.perspectivev3
+!default.perspectivev3
+xcuserdata
+*.xccheckout
+*.moved-aside
+DerivedData
+*.hmap
+*.ipa
+*.xcuserstate
+
```

Comment:
Not really sure if I did this right but should add `.vscode` to remove personal settings.

```suggestion

# .vscode
.vscode

```

Reply:
Added the .vscode, .idea, and .prettierjs.js names in the gitignore files of both the 2 example folders I created.